.eslintrc

```diff
@@ -0,0 +1,257 @@
+parser: "babel-eslint"
```

Comment:
Yep, I intend to pull in eslint-config-defaults.  I do like my eslintrc file but it has been superseded :)